sdk/managednetworkfabric/Azure.ResourceManager.ManagedNetworkFabric/src/Customize/NetworkFabricInternetGatewayData.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using Azure.Core;
+using Azure.ResourceManager.ManagedNetworkFabric.Models;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.ManagedNetworkFabric
+{
+    /// <summary>
+    /// A class representing the NetworkFabricInternetGateway data model.
+    /// The Internet Gateway resource definition.
+    /// </summary>
+    public partial class NetworkFabricInternetGatewayData : TrackedResourceData
+    {
+
+        /// <summary> IPv4 Address of Internet Gateway. </summary>
+        public IPAddress IPv4Address { get; }
```

Comment:
Since there is a new property `IPV4Address` with string type, and this can't be represented by the new one. We should add EB.Never and obsolete attribute to it.

Reply:
Updated as suggested.